apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -0,0 +1,194 @@
+/** Workshop Survey page. Displays survey results. ...*/
+import $ from 'jquery';
+import _ from 'lodash';
+import React from 'react';
+import {
+  Row,
+  Col,
+  Button
+} from 'react-bootstrap';
+import moment from 'moment';
+import 'react-select/dist/react-select.css';
```

Comment:
Is this needed? It doesn't look like you're importing the associated 'react-select` control that uses this css.